packages/typespec-test/test/openai_modular/spec/models/completions/chat_messages.tsp

```diff
@@ -110,6 +110,7 @@ model ChatRequestUserMessage extends ChatRequestMessage {
   @doc("The chat role associated with this message, which is always 'user' for user messages.")
   role: ChatRole.user;
 
+  @added(ServiceApiVersions.v2024_02_15_Preview)
   @doc("The contents of the user message, with available input types varying by selected model.")
```

Comment:
Could you share the link for ChatRequestUserMessage and ChatMessageContent in spec repo?

Reply:
`ChatRequestUserMessage`: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/chat_messages.tsp#L109

`ChatMessageContent`: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/chat_messages.tsp#L17

Reply:
@timotheeguerin our smoke testing failed with errors `@typespec/versioning/incompatible-versioned-reference` and we re-produced this [here](https://github.com/Azure/azure-rest-api-specs/pull/30407). I guess it is introduced by [pr](https://github.com/microsoft/typespec/pull/4179).

Could you help on the fix in spec repo?

Reply:
The link you gave in the spec repo fails before in the install step

Reply:
Also didn't had to update typespec-next for this one. Are you sure you are up to date with the space to start with(another reason why just blindly copying spec from the spec repo for test is not great)

Reply:
You can check the `typespec-next` branch with fixes for the next release this is the one you are missing 
<img width="1469" alt="image" src="https://github.com/user-attachments/assets/f4abf6c8-3a91-494b-9e9d-ac9c9ef817b1">